### DISCUSSION TITLE
Translate

### DISCUSSION THREAD

**Discussion by andriux26** [jekhokie/raspberry-noaa-v2#356] [General]
Helo how to translate  this   Southbound and Northbound on  the web i want Southbound to  -= Pietus =-    and Northbound to -=Siaure=-

**Comment by dom-robinson** [comment]
Hi - it is possible to edit / add 'languages' here >` https://github.com/jekhokie/raspberry-noaa-v2/tree/master/webpanel/App/Lang `

On your local machine the webpanel 'template' is the one in the raspberry-noaa-v2 folder, and when you run the ./install_and_upgrade.sh script it will replicate the webpanel to /var/www/wx-new/

So the 'live' language listings are (typically)  in `/var/www/wx-new/App/Lang` and you will need to edit the language you are set to (en.php by default)...

Hope that helps :)
